### DISCUSSION TITLE
Pricing Web3Auth SDK Modal & Base

### DISCUSSION THREAD

**Discussion by caesarvdidsert** [Web3Auth/Web3Auth#1093] [Help]
What is the pricing of the SDK Modal <1000 MAU?
What is the pricing of the SDK Base <1000 MAU?

and what is the difference between the Base & the Model

**Comment by shahbaz17** [comment]
Hey @caesarvdidsert 

We have a standard pricing plan and don't have separate pricing plans for different SDKs.

Please check out https://web3auth.io/pricing.html

**Reply by caesarvdidsert** [reply]
@shahbaz17 What is the most effective way to learn how to implement Web3Auth without any previous coding experience?

Is it recommended to do buy a JavaScript Basic Knowledge Course, or is there something else…
